ui-tests/src/test/java/org/openmrs/reference/CheckModulesTest.java

```diff
@@ -15,7 +15,7 @@
     /**
      * Check the list of modules to be sure they are all started.
      */
-    @Ignore
+    @Test
```

Comment:
This is a good start @insookwa  , Can you include  @Category(BuildTests.class) on top of the test

Reply:
the @category is just below the @Test

Reply:
https://github.com/openmrs/openmrs-contrib-qaframework/compare/master...insookwa:RATEST-149?expand=1